src/ripple/app/ledger/LedgerConsensus.h

```diff
@@ -56,16 +56,16 @@ class LedgerConsensus : public Traits
 
     virtual LgrID_t getLCL () = 0;
 
-    virtual void gotMap (TxSet_t const& map) = 0;
+    virtual void gotMap (Time_t const& now, TxSet_t const& map) = 0;
```

Comment:
On further thought, all the instances of the `NetClock` times are really the next close time are provided as an argument to public methods of `Consensus`.  So I can do as you suggest and drop the customization point.  The unit test can deal with network times as it wants to by converting values  `manual_clock<std::chrono::steady_clock>` to `NetClock::time_point`.  In the future, it could model the network clock drift explicitly if helpful.